### DISCUSSION TITLE
how make llamacpp inference fast as possible on SNAPDRAGON732G termux

### DISCUSSION THREAD

**Discussion by akac97** [ggml-org/llama.cpp#9327] [Q&A]
i want to run phi-3.5-mini-instruct gguf fast as possible on my phone what flags can i use llamacpp ./llama-cli if possible i want use q3_k_l or q3_k_m or q4_k_m

**Comment by ExtReMLapin** [comment]
For arm you can use the Q4_0_4_4 quants but does Kryo 470 counts as ARM ?

**Reply by akac97** [reply]
i think it counts as arm But i think about thinks like batch processing and something like that. (cli flags)

**Reply by ExtReMLapin** [reply]
You could use QNN backend from there https://github.com/chraac/llama.cpp/tree/dev-refactoring

It's originated from this PR, but it's a mess, read the comments https://github.com/ggerganov/llama.cpp/pull/6869
